plugin.test.js

```diff
@@ -0,0 +1,69 @@
+const axios = require('axios').default;
+const path = require('path');
+const gateway = require('express-gateway');
+const express = require('express');
+
+let Application = undefined;
+
+beforeAll((done) => {
+  axios.defaults.baseURL = 'http://localhost:8080/';
```

Comment:
You should consider using `axios.create` instead of changing configuration of default instance of axios.

Reply:
@stanislaw-glogowski Done